pyfifotax/historic_price_utils.py

```diff
@@ -0,0 +1,140 @@
+import os
+import datetime
+import decimal
+import pandas as pd
+import yfinance as yf
+import requests_cache
+from pathlib import Path
+
+
+def modify_history(hist: pd.DataFrame):
+    hist = hist.reset_index()
+    hist = hist[["Date", "Low", "High", "Open", "Close"]]
+    return hist
+
+
+def modify_splits(splits: pd.DataFrame, min_date: datetime.date):
+    splits[min_date] = 1.0
+    splits = splits.sort_index()
+    return splits
+
+
+def modify_history_and_splits(hist: pd.DataFrame, splits: pd.DataFrame):
+    hist = modify_history(hist)
+    min_date = hist.Date.min()
+    splits = modify_splits(splits, min_date)
+    return hist, splits
+
+
+def get_splits_from_ticker(ticker: str):
+    cache = requests_cache.CachedSession(
+        cache_name=os.path.join(
+            f"{Path.home()}", ".cache", "pyfifotax", "yfinance-cache"
+        ),
+        backend="sqlite",
+    )
+    stock = yf.Ticker(ticker, session=cache)
+    splits = stock.splits
+    return splits
+
+
+def get_history_and_splits_from_ticker(ticker: str):
```

Comment:
When is this function called exactly? After I put a log in the beginning, I don't see it in my debug output but it's possible that my use-case avoided it.